fe/fe-core/src/main/java/com/starrocks/sql/analyzer/ExpressionAnalyzer.java

```diff
@@ -1019,6 +1019,15 @@ public Void visitFunctionCall(FunctionCallExpr node, Scope scope) {
                 argumentTypes = node.getChildren().stream().map(Expr::getType).toArray(Type[]::new);
             } else if (DecimalV3FunctionAnalyzer.argumentTypeContainDecimalV2(fnName, argumentTypes)) {
                 fn = DecimalV3FunctionAnalyzer.getDecimalV2Function(node, argumentTypes);
+            } else if (FunctionSet.CARDINALITY.equals(fnName)) {
```

Comment:
you lose regesite `cardinality` signature? and don't need speical the implement in here

Reply:
ok